lms/djangoapps/verify_student/tests/test_models.py

```diff
@@ -748,6 +748,42 @@ def test_add_status_from_checkpoints(self, status):
             list(self.check_point2.checkpoint_status.all().values_list('location_id', flat=True))
         )
 
+    def test_get_location_id(self):
+        """ Getting location id for a specific checkpoint  """
+
+        # creating software secure attempt against checkpoint
+        self.check_point1.add_verification_attempt(SoftwareSecurePhotoVerification.objects.create(user=self.user))
+
+        # add initial verification status for checkpoint
+        VerificationStatus.add_verification_status(
+            checkpoint=self.check_point1,
+            user=self.user,
+            status='submitted',
+            location_id=self.dummy_reverification_item_id_1
+        )
+
+        attempt = SoftwareSecurePhotoVerification.objects.filter(user=self.user)
+
+        self.assertIsNotNone(VerificationStatus.get_location_id(attempt))
+        self.assertEqual(VerificationStatus.get_location_id(272), '')
```

Comment:
Replace `272` with something that makes sense:
`'i4x://InvalidOrg/InvalidCourse/edx-reverification-block/InvalidBlock'`